Keanu/src/app/app.component.ts

```diff
@@ -2,9 +2,13 @@ import { Component, ViewChild } from '@angular/core';
 import { Nav, Platform } from 'ionic-angular';
 import { StatusBar, Splashscreen } from 'ionic-native';
 
+// Default Pages (Leave alone for now)
 import { Page1 } from '../pages/page1/page1';
```

Comment:
Weird error when removing Page 1 and Page 2. Main.js would complain and break the program. Did this temp fix for now.